x-pack/packages/kbn-entities-schema/src/schema/patterns.ts

```diff
@@ -30,7 +30,8 @@ export function entitiesIndexPattern<TDataset extends Dataset>({
   dataset,
   definitionId,
 }: IndexPatternOptions<TDataset>) {
-  return `.${ENTITY_BASE_PREFIX}.${schemaVersion}.${dataset}.${definitionId}` as const;
+  // return `.${ENTITY_BASE_PREFIX}.${schemaVersion}.${dataset}.${definitionId}` as const;
+  return `${ENTITY_BASE_PREFIX}.${schemaVersion}.${dataset}.${definitionId}` as const;
```

Comment:
Why did we remove the `.` prefix?

Reply:
It was just for being able to see those indices while debugging, with `.` the indices are hidden.